eokwukwe/ecommerce-shop-api
# #170580481 Add Attribute Values

#### What does this PR do?
- Admin can add an attribute value
 #### Description of Task to be completed?
- [x] Admin can add an attribute value
 #### How should this be manually tested?
- clone the branch
- Run `npm install` to install dependencies
- Run `npm run dev` to start the development server
- Log in as an admin to obtain your access token to be sent along in the header as `USER_KEY: Bearer access_token`
- Send a `POST` request to `http://localhost:8000/api/v1/attributes/:attribute_id/values` with the following credentials:
```
{
  "value": "Wide"
}

NOTE: value is a required field
```
- Expected output if the inputs are correct: 
```
{
  "status": 201,
  "message": "Atrribute value added successfully"
}
```
- Run `yarn test` or `npm test` to test locally

 #### What are the relevant pivotal tracker stories?
[#170580481](https://www.pivotaltracker.com/story/show/170580481)
